src/tokens/aoTokens/ao.ts

```diff
@@ -350,6 +383,31 @@ export const createDataItemSigner =
     };
   };
 
+export const createDataItemKeystoneSigner =
+  (keystoneSigner: KeystoneSigner) =>
+  async ({
+    data,
+    tags = [],
+    target,
+    anchor,
+  }: {
+    data: any;
+    tags?: { name: string; value: string }[];
+    target?: string;
+    anchor?: string;
+  }): Promise<{ id: string; raw: ArrayBuffer }> => {
+    const signer = keystoneSigner;
+    const dataItem = createData(data, signer, { tags, target, anchor });
```

Comment:
@7i7o Once https://github.com/wanderwallet/Wander/pull/973 is merged, I will update this as well to add an anchor if anchor passed is `undefined` and remove this [tag](https://github.com/wanderwallet/Wander/pull/966/files#diff-619637a129734fa4c3a23b1e5553bfe94c38bbb746ac794433341d5356ecce53R246) as it will be unnecessary then.